### PR TITLE
Upgrade packages in Apis.GraphQl

### PR DESCRIPTION
**Pull request by agriffard**
Update https://www.npmjs.com/package/graphiql to 1.0.3

### CODE REVIEW

**Comment by agriffard** [comment]
I had to change the file node_modules\graphql-language-service-utils\esm\validateWithCustomRules.js to point to the correct rules.

**Comment by agriffard** [comment]
There is an error using graphiql in admin:
`Uncaught Error: Must provide Source. Received: undefined.`

File: src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/Assets/js/App.tsx [hunk 1]
```diff
@@ -123,7 +123,6 @@ function App() {
                 onToggleExplorer={handleToggleExplorer}
             />
             <GraphiQL
-                ref={ref => (_graphiql = ref)}
```
Comment (Jetski5822) [inline src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/Assets/js/App.tsx:126]: Are you sure this needs to be removed?

Comment (agriffard) [reply src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/Assets/js/App.tsx:126]: It is not working anymore with it.

No more `ref` property.

**Pull request merged**